modules/Book.js

```diff
@@ -0,0 +1,30 @@
+export default class Book {
+  constructor(title, author, id) {
+    this.title = title;
+    this.author = author;
+    if (id === undefined) {
+      this.createId();
+      return;
+    }
+    this.id = id;
+  }
+
+  createHtml() {
+    const book = document.createElement('div');
+    book.className = 'book';
+    book.innerHTML = `
+      <p>"<span>${this.title}</span>" by <span>${this.author}</span></p>
+      <button class="button--shadow" id="${this.id}">Remove</button>
+    `;
+    return book;
+  }
+
+  createId(tokenLen = 16) {
+    let id = '';
+    const chars = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789';
+    for (let i = 0; i < tokenLen; i += 1) {
+      id += chars.charAt(Math.floor(Math.random() * chars.length));
+    }
+    this.id = id;
+  }
```

Comment:
- [ ] @dfsalam great work on your functions, however as part of the requirement kindly change your function to an arrow function only the constructor method is exempted, its part of ES6 syntax.
![image](https://user-images.githubusercontent.com/61408860/212994577-8c8aa8dd-cb45-45b6-888d-0bcc74511418.png)